api_tests/users/views/test_user_detail.py

```diff
@@ -447,6 +448,27 @@ def user_one(self):
     def user_two(self):
         return AuthUserFactory()
 
+    @pytest.fixture()
+    def region(self):
+        return RegionFactory(name='Frankfort', _id='eu-central-1')
+
+    @pytest.fixture()
+    def region_payload(self, user_one, region):
+        return {
+            'data': {
+                'type': 'users',
+                'id': user_one._id,
+                'relationships': {
+                    'default_region': {
+                        'data': {
+                            'type': 'region',
```

Comment:
Should this be 'regions'?

Reply:
Yes, I can change to regions in my test, currently I don't believe type checking at this level has been implemented anywhere.

Reply:
Ember will fail in some instances when the types aren't correct, so it's good to be consistent. That was the reason for https://github.com/CenterForOpenScience/osf.io/pull/8917 for example. As long as the API responds with the right types, that's all I'm concerned about at this point. Incoming requests to the API don't need to be as fussy so far.

Reply:
gotcha, thanks for the context